### DISCUSSION TITLE
Docker buildx production ready status

### DISCUSSION THREAD

**Discussion by SantoshK07bec** [docker/buildx#1654] [General]
I have been using docker buildx for some time now with my docker desktop and it's a very handy tool for building multi-arch images. Now I am trying to use it in production system as well by integrating it with my Jenkins pipeline. However before I do that, I was trying to find if docker buildx on Linux is production ready tool as most of the place it is recommended to use with Docker Desktop for Development? I couldn't find any official documentation around it. Can someone clarify this please?

**Comment by polarathene** [comment]
As of v23 release earlier this year with Docker Engine, I think it's in a good state? Might be more to do with the updated BuildKit, but they also switched to `buildx` as the default IIRC, you should be able to confirm by looking up release notes for Docker Engine v23.

Many projects on Github that use the Github Actions CI also have been using buildx to build images for a while. I know a popular project that I help maintain has been. Works great for building images :+1: 

---

If you're building multi-platform images, be careful of running into [this problem with `FROM --platform`](https://github.com/docker/buildx/issues/844).
